eva-accession-import/src/main/java/uk/ac/ebi/eva/accession/dbsnp/io/DbsnpVariantsWriter.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.accession.dbsnp.io;
+
+import org.springframework.batch.item.ItemWriter;
+import org.springframework.data.mongodb.core.MongoTemplate;
+
+import uk.ac.ebi.eva.accession.dbsnp.persistence.DbsnpVariantsWrapper;
+
+import java.util.List;
+
+public class DbsnpVariantsWriter implements ItemWriter<DbsnpVariantsWrapper> {
```

Comment:
Should this be `MongoItemWriter` or `ItemStreamWriter`?

Reply:
neither, I think. this will delegate in the 3 writers listed. The point of extending `MongoItemWriter<T>` is to call a couple of setters in the constructor and it just works for T objects, but we won't write DbsnpVariantsWrappper objects.

Regarding the ItemStreamWriter, it's useful compared to ItemWriter if we need `open`, `close` and `update`; and I don't think we need them.